app/Filament/Resources/EventResource/RelationManagers/EventAwardsRelationManager.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Filament\Resources\EventResource\RelationManagers;
+
+use App\Filament\Actions\ProcessUploadedImageAction;
+use App\Filament\Enums\ImageUploadType;
+use App\Models\Event;
+use App\Models\EventAward;
+use App\Models\User;
+use Filament\Forms;
+use Filament\Forms\Form;
+use Filament\Resources\RelationManagers\RelationManager;
+use Filament\Tables;
+use Filament\Tables\Table;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Facades\Auth;
+
+class EventAwardsRelationManager extends RelationManager
+{
+    protected static string $relationship = 'awards';
+
+    public static function canViewForRecord(Model $ownerRecord, string $pageClass): bool
+    {
+        /** @var User $user */
+        $user = Auth::user();
+
+        return $user->can('manage', EventAward::class);
+    }
+
+    public function form(Form $form): Form
+    {
+        /** @var Event $event */
+        $event = $this->getOwnerRecord();
+        $nextTier = ($event->awards()->max('tier_index') ?? 0) + 1;
+
+        return $form
+            ->schema([
+                Forms\Components\TextInput::make('label')
+                    ->minLength(2)
+                    ->maxLength(40)
+                    ->required(),
+
+                Forms\Components\TextInput::make('achievements_required')
+                    ->numeric()
+                    ->required(),
```

Comment:
Let's say tier 1 requires 10 achievements. Should it be possible for tier 2 to require 5 achievements?

Reply:
No. How do we enforce that?

Reply:
This is what I came up with, give it a try:
```php
$minRequired = $event->awards()
    ->where('tier_index', '<', $nextTier)
    ->max('achievements_required') ?? 0;

return $form
    ->schema([
        Forms\Components\TextInput::make('label')
            ->minLength(2)
            ->maxLength(40)
            ->required(),

        Forms\Components\TextInput::make('achievements_required')
            ->numeric()
            ->rules([
                'required',
                'numeric',
                'min:' . ($minRequired + 1),
            ]),
```

Reply:
Expanded on this to set minimum _and_ maximum values to ensure values don't overlap.